src/views/dissolved-search/index.html

```diff
@@ -31,7 +31,7 @@ <h1 class="govuk-heading-xl">
             value: "alphabetical",
             text: "Show results alphabetically",
             hint: {
-                text: "Previous company names are currently not available for alphabetical search"
+                text: "Previous company names are currently not available for alphabetical Search"
```

Comment:
Is there a reason for `Search` being capitalised?

Reply:
Nope :) reoved